packages/components/package.json

```diff
@@ -14,6 +14,9 @@
     "@babel/preset-env": "^7.16.11",
     "@babel/preset-react": "^7.16.7",
     "@babel/preset-typescript": "^7.16.7",
+    "@mattermost/client": "*",
```

Comment:
This versioning is something I actually don't know how it works. I don't know if we have to specify specific versions or if we use * and NPM figures out what versions are required for the package when it's shipped on NPM

Reply:
if we really do need this stuff we should set them up as workspaces.